src/js/builtins/ProcessObjectInternals.ts

```diff
@@ -26,6 +26,7 @@
 
 export function getStdioWriteStream(fd) {
   const tty = require("node:tty");
+  const fs = require("node:fs");
```

Comment:
`require("node:fs")` is one of the most expensive imports we can do 

let's defer it as late as possible